mesheryctl/internal/cli/root/connections/connections.go

```diff
@@ -23,8 +23,14 @@ var ConnectionsCmd = &cobra.Command{
 	Short: "Manage connections",
 	Long:  `Manage connections`,
```

Comment:
It appears that between the short and the long description, one is not more descriptive than the other.

Reply:
Will update